### ISSUE TITLE
Colour contrast enhanced - DAC_Colour_Contrast_Enhanced_Issue1

### ISSUE THREAD

**Issue by gtvj** [nationalarchives/ds-wagtail#180]
On page 43 of the report DAC identify an Level AAA issue with ID `DAC_Colour_Contrast_Enhanced_Issue1`. They have explained that there are several places where the colour contrast meets the GOV.UK Design System, it does not meet Level AAA. 

We can fix this by reviewing all colour contrast used in the application to ensure Level AAA. 

### What was this not identified in the in-house testing
It is a Level AAA failure. We only tested against Level A and AA.
